client/src/main/java/io/spine/client/CommandRequest.java

```diff
@@ -171,13 +171,27 @@ public CommandRequest onServerError(ServerErrorHandler handler) {
      *         client code "knows" how many of them it expects. Also, some events may not arrive
      *         because of communication or business logic reasons. That's why the returned
      *         subscriptions should be cancelled by the client code when it no longer needs it.
+     * @see #postAndForget()
      */
-    @CanIgnoreReturnValue
     public ImmutableSet<Subscription> post() {
         PostOperation op = new PostOperation();
         return op.perform();
     }
 
+    /**
+     * Posts the command without subscribing to events that may be generated because of
+     * handling the command.
+     *
+     * @throws IllegalStateException
+     *   if {@link #observe(Class, EventConsumer)} or {@link #observe(Class, Consumer)} were
+     *   called in the command request configuration chain before calling this method
```

Comment:
Is it the new formatting style? We usually have the description aligned with the exception name